packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -475,6 +517,21 @@ class Transforms {
       return Promise.resolve(object);
     }
 
+    // decrypt description if present with a descriptionEncryptionKeyUrl
+    if (container.description && container.descriptionEncryptionKeyUrl) {
+      requestWithRetries(ctx.webex.internal.encryption, ctx.webex.internal.encryption.decryptText,
+        [container.descriptionEncryptionKeyUrl, container.description, {onBehalfOf: container.onBehalfOfUser}])
+        .then((decryptedContainerDescription) => {
+          container.description = decryptedContainerDescription;
+        })
+        .catch((reason) => {
+          ctx.webex.logger.error(`Decrypt container description error for ${container.containerType} container ${container.containerId}: ${reason}`);
+          // add warn property to container info - this warning will be recorded in the downloader
+          container.warning = reason;
+          // don't return, attempt to decrypt the name first
```

Comment:
I believe this is the correct thing to do here

Reply:
I don't think it matters. The call to `ctx.webex.internal.encryption.decryptText` _may_ need to retrieve a key. As such the `then` here occurs in the future. i.e. You've already executed the `containerName` code below.

It seems a bit wrong that this code only `returns` the `containerName` decryption promise. i.e. Technically it should return a `Promise.all(.....)` contain every promise here.

But I'm rusty on JavaScript at this point. I think I'm right but there is room for me to be wrong. Also that is not new to this PR.